lib/cli/run.js

```diff
@@ -87,7 +87,7 @@ exports.builder = yargs =>
       },
       extension: {
         default: defaults.extension,
-        defaultDescription: 'js',
+        defaultDescription: 'js,cjs,mjs',
```

Comment:
Remove this line, it's not necessary (see watch-ignore for example).

Reply:
Why is it not necessary? I looked at `watch-ignore` but couldn't understand why this was so. Don't we want to say that the default extensions we look at are those?

Reply:
`default` property does this job already, `defaultDescription` just overwrites it. If you have a look at the `--help` output, `--watch-ignore` shows everything as needed.

Reply:
You have to remove the complete line: `defaultDescription: 'js',` otherwise it overwrites `defaults.extension`.